docs/user-interface/images/app-icons.md

```diff
@@ -1,83 +1,215 @@
+> [!CAUTION]
+> If you don't clean the project and uninstall the app from the target platform, you may not see your new icon.
+
+After changing the icon, review the [Platform specific configuration](#platform-specific-configuration) information.
 
-The base size of the app icon can be specified by setting the `BaseSize` attribute to values that are divisible by 8:
+### Composed icon
+
+Alternatively, the app icon can be composed of two images, one image representing the background and another representing the foreground. Since icons are transformed into png files, the composed app icon will be first layered with the background image, typically an image of a pattern or solid color, followed by the foreground image. In this case, the `Include` attribute represents the icon background image, and the `Foreground` attribute represents the foreground image:
 
 ```xml
-<MauiIcon Include="Resources\AppIcon\appicon.jpg" BaseSize="64,64" />
+<ItemGroup>
+    <MauiIcon Include="Resources\AppIcon\appicon.svg" ForegroundFile="Resources\AppIcon\appiconfg.svg" />
+</ItemGroup>
 ```
 
-The value of the `BaseSize` attribute represents the baseline density of the image, and is effectively the 1.0 scale factor for the image from which all other density sizes are derived. This value will be used to ensure that the app icon is correctly resized to different display densities. If you don't specify a `BaseSize` for a bitmap-based app icon, the image isn't resized. If you don't specify a `BaseSize` value for a vector-based app icon, the dimensions specified in the SVG are assumed to be the base size. To stop vector images being resized, set the `Resize` attribute to `false`:
+> [!IMPORTANT]
+> The background image (`Include` attribute) must be specified for the `<MauiIcon>` item. The foreground image (`ForegroundFile` attribute) is optional.
+
+## Set the base size
+
+.NET MAUI uses your icon across multiple platforms and devices, and does its best to resize the icon according to those platforms and devices. The app icon is also used to represent different things, such as a store entry for your app or the icon used to represent the app after it's installed on a device.
+
+The base size of your icon represents baseline density of the image, and is effectively the 1.0 scale factor that all other sizes are derived. If you don't specify the base size for a bitmap-based app icon, such as a PNG file, the image isn't resized. If you don't specify the base size for a vector-based app icon, such as an SVG file, the dimensions specified in the image are used as the base size. To stop a vector image from being resized, set the `Resize` attribute to `false`.
+
+The following figure illustrates how base size affects an image:
+
+:::image type="content" source="media/app-icons/base-size.png" alt-text="How base size affects an app icon for .NET MAUI":::
```

Comment:
I went for this image because the actual icon for .NET MAUI is pretty boring 😁 

I changed the image text and article text to use 424 as the width, making it all divisible by 8.